channels/us.m3u

```diff
@@ -6187,8 +6179,6 @@ https://a.jsrdn.com/broadcast/4hhfi556/indexR720P.m3u8
 https://streams.the6tv.duckdns.org:2443/locals/BayArea/kfsf-dt1.m3u8
 #EXTINF:-1 tvg-id="KDTV.us" tvg-name="Univision (KDTV)" tvg-country="US" tvg-language="English" tvg-logo="https://upload.wikimedia.org/wikipedia/en/f/f6/KDTV-14_logo.png" group-title="Local",Univision (KDTV)
 https://streams.the6tv.duckdns.org:2443/locals/BayArea/kfsf-dt2.m3u8
-#EXTINF:-1 tvg-id="WUVP.us" tvg-name="WUVP-DT1" tvg-country="US" tvg-language="US" tvg-logo="https://static.locastnet.org/logo/504/1613144123818_UNI.png" group-title="",Univision Philadelphia (WUVP)
-https://cdn007.the6tv.duckdns.org:2443/the6tv-phl/wuvp-dt1/index.m3u8
```

Comment:
Only @MapGuy11 can answer why this channel is offline.

Reply:
Yeah With the Univision West feed above I felt that the service wasn't needed anymore. If I find a Univision East Feed I will add that too the list.